io_context_test.go

```diff
@@ -1,27 +1,133 @@
 package astiav_test
 
 import (
-	"io/ioutil"
+	"bytes"
+	"io"
 	"os"
 	"path/filepath"
+	"strings"
 	"testing"
 
 	"github.com/asticode/go-astiav"
 	"github.com/stretchr/testify/require"
 )
 
-func TestIOContext(t *testing.T) {
```

Comment:
As the IOContext tests are not that big, can you avoid splitting them into separate functions?

For instance you could:

- Write to the file using the IOContext => check error + n
- Seek using the IOContext
- Read from the file using the IOContext => check error + n + string == "testing"

Also, could you test `Seekable` as well?

Reply:
Sure, but concerning `Seekable`; im not sure if what case it would be false. Let me know if you have any suggestions for that.

Reply:
I think testing it's `true` when opening a file is sufficient 👍